### PR TITLE
Support qwen2.5 3B for NPU & update related examples

### PR DESCRIPTION
**Pull request by rnwang04**
### 1. Why the change?

https://github.com/analytics-zoo/nano/issues/1754#issuecomment-2496606222
Work with https://github.com/intel-analytics/llm.cpp/pull/678

### 2. User API changes

No change.

### 3. Summary of the change 

- Fix runtime error of qwen2.5 3B when running python acc lib
- Considering precision issue, default use CW int8 for qwen2.5 3B when running python acc lib / python pipeline / C++ NPU
- Update related example & readme
- Replace `load_in_low_bit` with `low_bit` parameter in example script

### 4. How to test?
- [ ] Unit test: Please manually trigger the PR Validation [here](https://github.com/intel-analytics/ipex-llm-workflow/actions/workflows/llm-PR-validation.yml) by inputting the PR number (e.g., `1234`). And paste your action link here once it has been successfully finished.

### CODE REVIEW

File: python/llm/example/NPU/HF-Transformers-AutoModels/LLM/qwen.py [hunk 1]
```diff
@@ -47,7 +47,10 @@
                         help='Prompt to infer')
     parser.add_argument("--n-predict", type=int, default=32, help="Max tokens to predict")
     parser.add_argument("--max-context-len", type=int, default=1024)
-    parser.add_argument("--max-prompt-len", type=int, default=512)
+    parser.add_argument("--max-prompt-len", type=int, default=960)
+    parser.add_argument("--quantization_group_size", type=int, default=0)
+    parser.add_argument('--load_in_low_bit', type=str, default="sym_int4",
```
Comment (jason-dai) [inline python/llm/example/NPU/HF-Transformers-AutoModels/LLM/qwen.py:52]: just "low_bit"?

File: python/llm/example/NPU/HF-Transformers-AutoModels/LLM/CPP_Examples/README.md [hunk 2]
```diff
@@ -35,9 +35,26 @@ pip install transformers==4.45.0 accelerate==0.33.0
...
 ```cmd
-:: to convert Qwen2.5-7b-Instruct
+:: to convert Qwen2.5-7B-Instruct
 python convert.py --repo-id-or-model-path Qwen/Qwen2.5-7B-Instruct --save-directory <converted_model_path>
 
+:: to convert Qwen2-1.5B-Instruct
+python convert.py --repo-id-or-model-path Qwen/Qwen2-1.5B-Instruct --save-directory <converted_model_path>
+
+:: to convert Qwen2.5-3B-Instruct
+python convert.py --repo-id-or-model-path Qwen/Qwen2.5-3B-Instruct --save-directory <converted_model_path> --load_in_low_bit "sym_int8"
```
Comment (jason-dai) [inline python/llm/example/NPU/HF-Transformers-AutoModels/LLM/CPP_Examples/README.md:45]: just "low_bit"?

File: python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md [hunk 3]
```diff
@@ -122,6 +123,9 @@ python llama.py --repo-id-or-model-path meta-llama/Llama-3.2-3B-Instruct
 :: to run Qwen2-1.5B-Instruct (LNL driver version: 32.0.101.2715)
 python qwen.py
 
+:: to run Qwen2.5-3B-Instruct (LNL driver version: 32.0.101.2715)
+python qwen.py --repo-id-or-model-path Qwen/Qwen2.5-3B-Instruct --load_in_low_bit sym_int8
```
Comment (jason-dai) [inline python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md:127]: just "low_bit"?

File: python/llm/example/NPU/HF-Transformers-AutoModels/LLM/qwen.py (hunk 1 above)
Comment (rnwang04) [reply python/llm/example/NPU/HF-Transformers-AutoModels/LLM/qwen.py]: done

File: python/llm/example/NPU/HF-Transformers-AutoModels/LLM/CPP_Examples/README.md (hunk 2 above)
Comment (rnwang04) [reply python/llm/example/NPU/HF-Transformers-AutoModels/LLM/CPP_Examples/README.md]: done

File: python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md (hunk 3 above)
Comment (rnwang04) [reply python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md]: done

**Review by jason-dai — APPROVED** [review]
LGTM

**Pull request merged**